tools/deploy/deploy.go

```diff
@@ -71,35 +70,23 @@ func parseConfig() (c config) {
 	return c
 }
 
-// updateChefRepo ensures the lates chef cookbooks are pulled before deploying.
-// Checks github first and ignores pull if already up to date.
+// Update ChefRepo to ensure the latest chef cookbooks are pulled before deploying.
+// Current implementation is Gengo specified. Please re-implement this function according to your environment
 func updateChefRepo(conf config) {
-	githubToken := os.Getenv(gitHubAPITokenEnvVar)
-	t := &oauth.Transport{
-		Token: &oauth.Token{AccessToken: githubToken},
-	}
-	client := github.NewClient(t.Client())
-	s := "git --git-dir=" + conf.chefRepo + "/.git rev-parse HEAD"
-	localHash, _ := execCmd(s, conf)
-	commits, _, err := client.Repositories.ListCommits("Gengo", "devops-tools", nil)
+	log.Println("Updating devops-tools")
+	os.Setenv("GIT_SSH", "/tmp/private_code/wrap-ssh4git.sh")
+	gitCheckoutCmd := "/usr/bin/git --git-dir=" + conf.chefRepo + "/.git --work-tree=" + conf.chefRepo + " checkout " + *deployToolBranch
+	// TODO: refactor "execCmd" and run commands at once
+	_, err := execCmd(gitCheckoutCmd, conf)
 	if err != nil {
-		log.Fatal("ERROR:  failed to get commits from GitHub: Please try again later ", err)
+		log.Fatal("ERROR: Failed to checkout: ", err)
 	}
-	remoteHash := *commits[0].SHA
```

Comment:
Removed comparison with remote branch because always pulling the latest would work.